pages/wi/partners.js

```diff
@@ -106,7 +106,11 @@ const partnerListing = () => {
       <ContentSection>
         <Grid columns="repeat(auto-fit,minmax(32rem,1fr))">
           <Cell>
-            <Header>2019 Sponsors & Partners</Header>
+            <Header>
+              2019
+              <br />
+              Partners
+            </Header>
```

Comment:
When we need to wrap a header we will adjust the width and let it wrap naturally. Line breaks have a tendency to make this look off when screen sizes are adjusted so we avoid it as much as possible. I think in this case it is a matter of adjusting the graphic size, location, spacing, all that. 

We could add this as an open source issue, and I'm sure a bunch on contributors would happily pick it up and start playing with the graphic and responsiveness here.

Reply:
Yeah, the line break felt really wrong, so I figured one of you would set me straight!!

Reply:
Haha, I know how tempting there are 😄 
For this I say just update it to "2019 Partners" and let's get it in. Good catch that we had "sponsors" still here. 

Then if the graphic is still off on the various size screens we open a Github issue for the community to pick up.

Reply:
Corrected Sara